apps/condo/cypress/e2e/pages/auth.cy.js

```diff
@@ -1,32 +1,48 @@
 import { faker } from '@faker-js/faker'
 
 import { SignIn, ForgotPassword, Registration }  from '../../objects/Auth'
+import { SimpleTracer } from '../../objects/helpers'
 
-describe('Auth scenarios', () => {
+describe.skip('Auth scenarios', () => {
```

Comment:
xD

Reply:
I'm going to stop running Auth tests until we figure out what to do with foreign numbers and SMS messages

Reply:
May be you need skipIf condition?